src/app/(proper_react)/(redesign)/(public)/LandingView.tsx

```diff
@@ -31,231 +31,236 @@ import { FaqSection } from "./Faq";
 import { SignInButton } from "../../../components/client/SignInButton";
 import { RebrandAnnouncement } from "./RebrandAnnouncement";
 import { FeatureFlagName } from "../../../../db/tables/featureFlags";
+import { GaScript } from "../GaScript";
 
 export type Props = {
   eligibleForPremium: boolean;
   l10n: ExtendedReactLocalization;
   countryCode: string;
   scanLimitReached: boolean;
   enabledFlags: FeatureFlagName[];
+  nonce: string;
 };
 
 export const View = (props: Props) => {
   return (
-    <main className={styles.wrapper}>
-      <nav className={styles.nav}>
-        <h1>{props.l10n.getString("public-nav-name")}</h1>
-        <SignInButton />
-      </nav>
-      <header className={styles.hero}>
-        <div className={styles.heroContent}>
-          <h1>{props.l10n.getString("landing-all-hero-title")}</h1>
-          <p>
-            {props.l10n.getString(
-              props.eligibleForPremium
-                ? "landing-premium-hero-lead"
-                : "landing-all-hero-lead",
+    <>
+      <GaScript nonce={props.nonce} />
```

Comment:
We don’t need `GaScript` anymore if we use the `GoogleAnalytics` component from Next.js in `layout.tsx`.
```suggestion
```